src/charts/trends/trends-chart.html

```diff
@@ -19,7 +19,7 @@
   </div>
   <div ng-switch-when="compact" class="trend-card-compact-pf">
     <div class="row trend-row">
-      <div class="col-sm-4 col-md-4">
+      <div class="col-md-2">
```

Comment:
I believe you wanted col-sm-2 here so that at small resolutions these stay on a single line. Only at xs should they be vertical. This goes for all the changes here.